tests/test_controls.py

```diff
@@ -329,25 +329,25 @@ def test_notify(self):
             {
                 "type": "request",
                 "uri": "ssap://system.notifications/createToast",
-                "payload": {"message": "test", "iconData": None, "iconExtension": "png"},
+                "payload": {"message": "test", "iconData": None, "iconExtension": None}
             }
         )
 
-    def test_notify__icon(self):
+    def test_notify_icon(self):
         def mock_get_icon_data(*args, **kwargs):
             return "dummy.png"
 
         pywebostv.controls.get_icon_data = mock_get_icon_data
 
```

Comment:
Can we please remove these unused patching steps?

Reply:
whoops, done.